pkg/executables/kind.go

```diff
@@ -50,6 +50,7 @@ type kindExecConfig struct {
 	RegistryCACertPath     string
 	ExtraPortMappings      []int
 	DockerExtraMounts      bool
+	CloudStackExtraMounts  []string
```

Comment:
Why does Kind have something Cloud Stack specific? Kind is just a development cluster management tool with some quirks around how it works. It has no reason to know about Cloud Stack.